src/Mvc/Mvc.Core/src/DependencyInjection/MvcCoreMvcBuilderExtensions.cs

```diff
@@ -85,7 +85,14 @@ public static IMvcBuilder AddApplicationPart(this IMvcBuilder builder, Assembly
                 throw new ArgumentNullException(nameof(assembly));
             }
 
-            builder.ConfigureApplicationPartManager(manager => manager.ApplicationParts.Add(new AssemblyPart(assembly)));
+            builder.ConfigureApplicationPartManager(manager =>
```

Comment:
This is odd. This change exists in master: https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/Mvc.Core/src/DependencyInjection/MvcCoreMvcBuilderExtensions.cs#L88-L95, but not in 2.2. Why does it show up in the diff?